pageserver/src/tenant/layer_map.rs

```diff
@@ -639,147 +639,10 @@ impl LayerMap {
         }
 
         println!("historic_layers:");
-        for layer in self.iter_historic_layers() {
-            layer.dump(verbose, ctx)?;
+        for desc in self.iter_historic_layers() {
+            desc.dump();
         }
         println!("End dump LayerMap");
         Ok(())
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
These were the "failure mode" of replacement work. All of the replacement functionality was removed. I added that "failure mode" body of work initially in #3513 but that was only needed because of RemoteLayer.